zingolib/src/wallet/keys/unified.rs

```diff
@@ -695,6 +627,15 @@ impl WalletCapability {
             },
         }
     }
+
+    pub(crate) fn ephemeral_ivk(
```

Comment:
I think its a mistake to have key related helper functions attached to WalletCapability, I think all key related stuff should stay contained in UnifiedKeyStore. UnifiedKeyStore has a TryFrom for AccountPubKey so I dont see why this helper is neccessary but I think it should at least be on UnifiedKeyStore

Reply:
Resolved by preceding comment.

Reply:
This function goes WalletCapability->AccountPubKey->EphemeralIvk. I was calling `AccountPubKey::try_from(wallet_capability().unified_key_store()).ephemeral_ivk()` in enough places I wanted to make a helper function for it.

Turning it into a unified_key_store method means that in all the places it's used, we'd need to add `.unified_key_store()` first...which isn't a very large cost, but this function exists to combine 3-ish lines of code to 1, and going down to 2 lines of code defeats half the point. 

What makes a helper 'necessary'? If you're worried about cluttering the `unified_key_store` impls, would moving it to be a free function instead of a method solve that?

Reply:
ok thats fine. I think i am anticipating that the keystore might move eventually so its best to keep it contained but we can cross that bridge later.